test_unstructured_ingest/unit/test_paths.py

```diff
@@ -23,15 +23,25 @@ class FakeConfigFolder:
     download_dir = "/fakeuser/fake_download"
 
 
+class DropboxIngestDocTest(DropboxIngestDoc):
+    def __post_init__(self):
+        return None
+
+
+class FsspecIngestDocTest(FsspecIngestDoc):
```

Comment:
nit: can we call these like `DropboxIngestTestDoc` or `TestDropboxIngestDoc`? rather than `DropboxIngestDocTest`. otherwise I'm expecting this class itself to be a test.